### DISCUSSION TITLE
Welcome to tutorials Discussions!

### DISCUSSION THREAD

**Discussion by likefallwind** [deepmodeling/tutorials#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ijazaskar** [comment]
hi, can you just give me few steps , how can i interconnect lammps and dpgen,  do i need to submit lammps input file or need to modify the existing lammps.py code , can you give me tille explanation , thanks
